src/ssl.c

```diff
@@ -9441,7 +9457,8 @@ int wolfSSL_CertManagerLoadCRLFile(WOLFSSL_CERT_MANAGER* cm, const char* file,
         return BAD_FUNC_ARG;
 
     if (cm->crl == NULL) {
-        if (wolfSSL_CertManagerEnableCRL(cm, 0) != WOLFSSL_SUCCESS) {
+        if (wolfSSL_CertManagerEnableCRL(cm, WOLFSSL_CRL_CHECK)
```

Comment:
I see this same wolfSSL_CertManagerEnableCRL(cm, 0) call in wolfSSL_CertManagerLoadCRLBuffer, does that not need to be changed too?